packages/web/components/layouts/main.tsx

```diff
@@ -34,6 +39,8 @@ export const MainLayout = observer(
       ({ selectionTest }) => selectionTest?.test(router.pathname) ?? false
     );
 
+    useHasAssetVariants();
```

Comment:
Is it necessary to check for this on every page? May affect performance for other pages. Why not where actionable, like on portfolio page? May be a question for nass